lib/web_ui/dev/watcher.dart

```diff
@@ -34,11 +34,14 @@ class Pipeline {
 
   Future<dynamic> _currentStepFuture;
 
-  PipelineStatus status = PipelineStatus.idle;
+  PipelineStatus get status => _status;
```

Comment:
nit: unnecessary refactoring?

Reply:
This is intentional. I wanted to make sure this property isn't publicly mutable.